addons/hr_contract/models/hr_contract.py

```diff
@@ -177,6 +177,16 @@ def _check_dates(self):
                     contract=contract.name, start=contract.date_start, end=contract.date_end,
                 ))
 
+    def _get_normalized_wage(self):
+        # This method is overriden in hr_payroll as without that module nothing allows to know
+        # at which frequency is paid the employee
```

Comment:
Is this comment referring to `_get_contract_wage` or `_get_normalized_wage`? It's a bit unclear right now. I'm guessing the former, in which case it should be a docstring